tests/study_tests/test_study.py

```diff
@@ -1577,3 +1578,24 @@ def test_set_invalid_metric_names() -> None:
     study = create_study(directions=["minimize", "minimize"])
     with pytest.raises(ValueError):
         study.set_metric_names(metric_names)
+
+def objective(trial):
```

Comment:
We can remove this function because we can use `func` defined in this file as an objective.